### PR TITLE
New tree

### LINKED ISSUE #167: new file tree
tree should be 
```
penn_chime/
| -app.py
| -test_app.py
| penn_chime/
  | -models.py
  | -presentations.py
  | ...
```

### PR DESCRIPTION
**Pull request by quinn-dougherty**
closes https://github.com/CodeForPhilly/chime/issues/167 

tree is now
```
src/
| -app.py
| -test_app.py
| penn_chime/
  | -models.py
  | -presentations.py
  | ...
```

### CODE REVIEW

**Comment by quinn-dougherty** [comment]
in https://github.com/CodeForPhilly/chime/pull/169/commits/026c47c3ee2aae7aee971afad1c3bbac3cc563b6 i updated dockerfile and docs

**Comment by jlubken** [comment]
It looks like the Procfile and Dockerfile don't reference src.

**Comment by jlubken** [comment]
Update paths in cli.py and maybe move it into src.

**Review by jlubken — APPROVED** [review]

File: docs/contributing/app-dev.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ If you need to run the application on a different port than the default (8000),
...
 ```
 
 ## Project Layout
 
 ### Application files
 
-- `app.py`: Main source for the application
-- `test_app.py`: [pytest](https://docs.pytest.org/en/latest/) tests for `app.py`
+- `penn_chime/app.py`: Main source for the application
+- `penn_chime/test_app.py`: [pytest](https://docs.pytest.org/en/latest/) tests for `app.py`
```
Comment (themightychris) [inline docs/contributing/app-dev.md:53]: @quinn-dougherty looks like we missed a spot

Comment (quinn-dougherty) [reply docs/contributing/app-dev.md:53]: thanks

**Pull request merged**